interface/backend/api/views.py

```diff
@@ -172,6 +173,21 @@ def get(self, request):
         return Response({'directories': tree})
 
 
+@api_view(['GET'])
+def candidates_info(request):
+    all_candidates = Candidate.objects.prefetch_related('case__series').all()
+    serialized_candidates = serializers.CandidateSerializer(all_candidates, context={'request': None}, many=True).data
+
+    # append DICOM files to response
+    for candidate in serialized_candidates:
+        series = candidate['case']['series']
+
+        if 'files' not in series:
+            series['files'] = glob.glob1(series['uri'], '*.dcm')
```

Comment:
Can you document why ``glob1`` and not regular ``glob``?

Reply:
I will add a comment. Btw, this entire piece should be moved from here. We should discover and save list of files only once - when creating a case. So as soon as #145 is done(PR #233 is pending), we will move it.